includes/class-wcpdf-admin.php

```diff
@@ -262,11 +260,8 @@ public function invoice_columns_data( $column ) {
 		}
 
 		$invoice = wcpdf_get_invoice( $order );
-		
-		if ( ! empty( $invoice ) ) {
-			$invoice_number = $invoice->get_number();
-			$invoice_date = $invoice->get_date();
-		}
+		$invoice_number = ! empty( $invoice ) && ! empty( $invoice->get_number() ) ? $invoice->get_number() : '';
+		$invoice_date = ! empty( $invoice ) && ! empty( $invoice->get_date() ) ? $invoice->get_date() : '';
```

Comment:
I would do this inside the switch statement, no need to retrieve the date if you're want to have the number for example.

Reply:
Done here: [8cfb4b0](https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/pull/412/commits/8cfb4b02d493cf79b5248953f102254d2c1f4ab9)